core/eds_test.go

```diff
@@ -38,13 +35,14 @@ func TestEmptySquareWithZeroTxs(t *testing.T) {
 		Txs: []types.Tx{},
 	}
 
-	eds, err := extendBlock(&data, appconsts.LatestVersion)
+	eds, err := extendBlock(&data)
 	require.NoError(t, err)
 	require.True(t, eds.Equals(share.EmptyEDS()))
 
 	// force extend the square using an empty block and compare with the min DAH
-	eds, err = app.ExtendBlock(data, appconsts.LatestVersion)
-	require.NoError(t, err)
+	// TODO(chatton): app.ExtendBlock was removed, can we delete?
```

Comment:
I'm unsure of what to do with this, `app.ExtendBlock` no longer exists.

Reply:
should be able to just do it in line

```go
dataSquare, err := square.Construct(
	data.Txs.ToSliceOfBytes(),
	appconsts.SquareSizeUpperBound,
	appconsts.SubtreeRootThreshold,
)

eds, err := da.ExtendSquare(share.ToBytes(dataSquare))
```

Reply:
that's basically already what happens in the unexported `extendBlock` fn, I would imagine there was an intentional reason to use both in the same test fn 🤔

Reply:
do you have any suggestions about how to do this? I'm not familiar enough with the code base yet to write this without some help. I can do in a follow up if that's okay?